content/src/licenses/architecture.md

```diff
@@ -7,21 +7,19 @@ previewType: renewal
 callToActionLink: https://newjersey.mylicense.com/eGov/Login.aspx
 callToActionText: Renew My Architect License
 ---
+
 ### Renewal Requirements
+
```

Comment:
[sand] `bodyHeading`?

Reply:
Good catch! License data won't be updated. I think the file only changed because it added spacing.